fedimint/src/bin/configgen.rs

```diff
@@ -5,22 +5,46 @@ use fedimint_api::{Amount, PeerId};
 use rand::rngs::OsRng;
 use std::path::PathBuf;
 
+/// Config generator for Fedimint Federation
+///
+/// Running this program will generate config
+/// files for federation member nodes in directory
+/// specified with `out-dir`
 #[derive(Parser)]
 struct Options {
-    cfg_path: PathBuf,
-    nodes: u16,
+    /// Directory to output all the generated config files
+    #[clap(long = "out-dir")]
+    dir_out_path: PathBuf,
+
+    /// Number of nodes in the federation
+    #[clap(long = "num-nodes")]
+    num_nodes: u16,
+
+    /// Base hbbft port
+    #[clap(long = "hbbft-base-port", default_value = "17240")]
```

Comment:
I checked that the port is out of the range for ephemeral ports on most systems (32768–60999 for Linux apparently).